{DB0:{Driver:sqlite3 Connection:./../database/person23.db Connection2:./../database/person23.db Connection3:./../database/allperson.db} Static:E:/dnapersondb/WebRoot LogPath:log Port:8080 Lims1LoginUrl:http://127.0.0.1:7070/dnalims/svc/login SqlTmplPath:Sqls CollectAgencyName:新疆公安局 CollectAgencyCode:650000123654 Code:001 Local:http://127.0.0.1:8080 Public:http://127.0.0.1:8088 Mode:local Server:650000 Updatedb:http://127.0.0.1:9999/api/downloaddb/}
Listening on 8080....
dial tcp 127.0.0.1:8088: connectex: No connection could be made because the target machine actively refused it.
goroutine 22 [running]:
web/middleware.RecoveryMiddleware.func1.1(0xb66460, 0xc042002400, 0xc042000500)
	E:/dnapersondb/src/web/middleware/recoveryMiddleware.go:24 +0xef
panic(0x8fb920, 0xc042010460)
	C:/go64/src/runtime/panic.go:489 +0x2dd
web/httpcontext.(*HttpContext).CheckWrite(0xc0420083f0, 0xb618e0, 0xc042010460)
	E:/dnapersondb/src/web/httpcontext/ctx.go:56 +0x2e7
service/card.(*Card).Geturlisornotonline(0xc0420083f0)
	E:/dnapersondb/src/service/card/card.go:724 +0x252
reflect.Value.call(0x951200, 0xc0420083f0, 0x2a13, 0x95f898, 0x4, 0xc042029820, 0x0, 0x0, 0xc042004040, 0x13, ...)
	C:/go64/src/reflect/value.go:434 +0x926
reflect.Value.Call(0x951200, 0xc0420083f0, 0x2a13, 0xc042029820, 0x0, 0x0, 0xc0420083f0, 0x2a13, 0x0)
	C:/go64/src/reflect/value.go:302 +0xab
register.MakeTypeHttpRouter.func1(0xb66460, 0xc042002400, 0xc042000500, 0xc0420023e0, 0x1, 0x1)
	E:/dnapersondb/src/register/util.go:31 +0x5a0
web/middleware.RecoveryMiddleware.func1(0xb66460, 0xc042002400, 0xc042000500, 0xc0420023e0, 0x1, 0x1)
[Golang]2017-06-12 09:39:56.2611314 +0800 CST | 200 |  13.0727477s | 127.0.0.1:8080 | POST | /api/Geturlisornotonline 
